lib/ex_okex/config.ex

```diff
@@ -2,6 +2,14 @@ defmodule ExOkex.Config do
   @moduledoc """
   Stores configuration variables for signing authenticated requests to API.
   """
+
+  @type t :: %ExOkex.Config{
+          api_key: String.t(),
+          api_secret: String.t() | nil,
```

Comment:
What is the case when this is `nil`?

Reply:
@yurikoval `sign_request` https://github.com/acuityinnovations/ex_okex/blob/182b36d180f612f3f865c2f90b9bf15e4fa17101/lib/ex_okex/api/private.ex#L53

Reply:
I'm trying to figure out if `lib/ex_okex/api/private.ex:53` is a mistake and shouldn't have fallback. I don't think there is a case when okex will have an empty key. Can be fixed in separate PR, though.

Reply:
Fwiw I planned on making this required in a separate PR I'm working on to DRY up the authentication logic. https://github.com/fremantle-capital/ex_okex/commit/46afafa241588acd88ea0dd7df4495b825238804